solution/build.sh

```diff
@@ -87,7 +87,9 @@ function copy_yamls()
     mkdir -p ${crd_dir} ${role_dir}
 
     kustomize build "${zenko_operator_repo}/config/artesca-solution/crd?ref=$(zenko_operator_tag)" -o ${crd_dir}
-    rename '.yaml' '_crd.yaml' ${crd_dir}/*
+    for file in ${crd_dir}/*.yaml ; do 
+        mv $file ${file%.yaml}_crd.yaml
+    done
```

Comment:
for those who don't have `rename`:  https://github.com/util-linux/util-linux

Reply:
worth noting the binary is actually called `rename.ul` in ubuntu, though..